android/src/main/kotlin/africa/ejara/trustdart/WalletHandler.kt

```diff
@@ -29,13 +29,13 @@ class WalletHandler {
             "NEAR"  to NEAR(),
             "XLM"   to XLM(),
             "BNB"   to BNB(),
-            "BSC"   to BSC(),
+            "BSC"   to ETH(),
             "DOGE"  to DOGE(),
-            "MATIC" to MATIC(),
+            "MATIC" to ETH(),
```

Comment:
Let's remove this then and just always use ETH . So all ETH compatible will just use the ETH